course/models.py

```diff
@@ -252,72 +301,97 @@ def __unicode__(self):
 # {{{ participation
 
 class ParticipationTag(models.Model):
-    course = models.ForeignKey(Course)
-
+    course = models.ForeignKey(Course,
+            verbose_name = 'Course ID')
```

Comment:
Sorry, I mix "identifier" with "ID", I had thought that they have the same meaning.